test/dialect/test_sqlite.py

```diff
@@ -2297,10 +2299,63 @@ def test_create_index_with_schema(self):
                     "unique": 0,
                     "name": "ix_main_l_bar",
                     "column_names": ["bar"],
+                    "dialect_options": {},
                 }
             ],
         )
 
+    def test_reflect_partial_indexes(self):
+        with testing.db.begin() as conn:
+            conn.exec_driver_sql(
+                "create table foo_with_partial_index (x integer, y integer)"
+            )
+            conn.exec_driver_sql(
+                "create unique index ix_partial on "
```

Comment:
**Federico Caselli** (CaselIT) wrote:

Done

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4200